apps/meteor/tests/data/livechat/rooms.ts

```diff
@@ -62,6 +62,22 @@ export const createVisitor = (department?: string): Promise<ILivechatVisitor> =>
 		});
 	});
 
+export const deleteVisitor = (token: string): Promise<ILivechatVisitor> =>
+	new Promise((resolve, reject) => {
+		request.get(api(`livechat/visitor/${token}`)).end((err: Error) => {
```

Comment:
Do we need this `get` call here? What is the purpose of that? If this is required somehow, we can use async await here instead of returning a promise